content/src/display-content/business-formation/nexus/authorize-business-entity.md

```diff
@@ -8,14 +8,12 @@ callToActionLink: https://www.nj.gov/treasury/revenue/pdf/2000.pdf
 callToActionText: Authorize My Business
 agencyId: nj-revenue-enterprise-services
 required: true
+summaryDescriptionMd: >-
+  You must register your business name and `authorize|certificate-formation` your
+  `business structure|business-structure-learn-more` with the State.
+bodyHeading: Application Requirements
 ---
 
```

Comment:
[dust]: Would this create different page content? With the previous ones the body header was above the summaryDescription but with this one it was below in the diff.